src/main/java/me/botsko/prism/database/PrismDataSourceUpdater.java

```diff
@@ -19,5 +19,7 @@
     void v6_to_v7();
 
     void v7_to_v8();
+
+    void v8_to_v9();
```

Comment:
Codacy found an issue: [The instance method name 'v8_to_v9' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5653372)